features/testbot/nil.feature

```diff
@@ -0,0 +1,42 @@
+@routing @testbot @nil
+Feature: Testbot - Check assigning nil values
+    Scenario: Assign nil values to all way strings
+        Given the profile file "testbot" extended with
+        """
+        function way_function (way, result)
```

Comment:
Yes this is well defined in any scripting language I can think of (javascript, python, ruby and lua): Last definition always overrides. Pretty similar to the name shadowing for variables. Only C++ and the likes are a little bit more picky.